docs/design/coreclr/botr/ryujit-overview.md

```diff
@@ -3,7 +3,9 @@ JIT Compiler Structure
 
 # Introduction
 
-RyuJIT is the code name for the next generation Just in Time Compiler (aka “JIT”) for the AMD64 .NET runtime. Its first implementation is for the AMD64 architecture. It is derived from a code base that is still in use for the other targets of .NET.
+RyuJIT is the code name for the next generation Just in Time Compiler (aka "JIT") for the .NET runtime. It was
```

Comment:
Nit: Is it still "the next generation"?  It's now been around for quite some time ;)